service/history/workflow/update/update_test.go

```diff
@@ -904,6 +905,81 @@ func TestWaitLifecycleStage(t *testing.T) {
 	})
 }
 
+func TestCompletedWorkflow(t *testing.T) {
+	var (
+		ctx     = context.Background()
+		effects = effect.Immediate(ctx)
+
+		meta = updatepb.Meta{UpdateId: t.Name() + "-update-id"}
+		req  = updatepb.Request{Meta: &meta, Input: &updatepb.Input{Name: t.Name()}}
+
+		store = mockEventStore{Controller: effects}
+	)
+
+	t.Run("request", func(t *testing.T) {
```

Comment:
this is more then "request". Could you clarify?

Reply:
Gave them a better name.